.chloggen/clickhouseexporter-persistentqueue-support.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: clickhouseexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note:  Add persistent storage support to clickhouse exporter
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [27653]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
Can you explain a bit here on how this becomes a breaking change?

Reply:
Thanks for pointing this out.  I had just taken the default config that the `exporterhelper.QueueSettings` provided, and thus the number of default consumers would have changed from 1 to 10.  However, that was a bit silly, so I've just kept the previous default in place.